openmc/statepoint.py

```diff
@@ -536,9 +567,15 @@ def read_source(self):
             # Store the source site in the NumPy array
             self._source[i] = site
 
-
     def compute_ci(self, confidence=0.95):
-        """Computes confidence intervals for each Tally bin."""
+        """Computes confidence intervals for each Tally bin.
```

Comment:
You might mention in a follow-on sentence that this routine is equivalent to calling ``compute_stdev(...)`` when the confidence is known as opposed to its corresponding t value. Just would be good to make it clear that calling one after the other will write over the results in ``std_dev`` from the first call.